dora/job/server/src/main/java/alluxio/job/plan/stress/StressBenchDefinition.java

```diff
@@ -88,14 +89,18 @@ public Set<Pair<WorkerInfo, ArrayList<String>>> selectExecutors(StressBenchConfi
     }
     workerList = workerList.subList(0, clusterLimit);
 
+    int index = 0;
     for (WorkerInfo worker : workerList) {
       LOG.info("Generating job for worker {}", worker.getId());
-      ArrayList<String> args = new ArrayList<>(2);
+      ArrayList<String> args = new ArrayList<>(4);
```

Comment:
Why we need to specify the initial capacity of the list? Is it for saving memory?

Reply:
yes the ArrayList will have capacity 10 if not specified. It's a good practice specifying the size if you know.